context/brand-context/springer/scss/10-settings/colors/default.scss

```diff
@@ -5,20 +5,38 @@
 
 $context--colors: (
 	'accent': #ee7d11,
+	'action-base': #004b83,
+	'action-light': scale-color(#004b83, $lightness: 10%),
+	'action-dull': #97bfd8,
+	'action-background-base': #ebf6ff,
+	'action-background-light': #f7fbfe,
+	'action-focus': #fc0,
+	'blue': #069,
+	'dark-blue': #173962,
 	'corporate': #00285a,
 	'corporate-light': scale-color(#00285a, $lightness: 10%),
+	'cta-top-color': #4d78af,
+	'cta-bottom-color': #3365a0,
+	'cta-hover-color': #0059c9,
 	'error': #c40606,
 	'warning': #fc0,
 	'info': #004b83,
 	'success': #71ab0b,
 	'visited': #a345c9,
-	'action-base': #004b83,
-	'action-light': scale-color(#004b83, $lightness: 10%),
-	'action-dull': #97bfd8,
-	'action-background-base': #ebf6ff,
-	'action-background-light': #f7fbfe,
-	'action-focus': #fc0,
-	'yellow': #ff9
+	'blueberry': #a6d8e7,
```

Comment:
The colours from line 26 onwards seem to be nature colours, is that intentional?

Reply:
I did find all those colours in https://github.com/springernature/oscar/blob/main/lib/frontend/sites/oscar-sites-springer/src/app-springerlink/scss/enhanced-article.scss
However by removing them individual from my local build I hope to have got rid off any that are not used.

Reply:
LGTM 😬